packages/lib/modules/vebal/useVebalUserData.ts

```diff
@@ -24,7 +24,7 @@ export function useVebalUserData() {
   return {
     data,
     refetch,
-    isConnected,
+    isConnected, // FIXME: [JUANJO] we shouldn't return account data from here
```

Comment:
Can you fix it already in this PR?

Reply:
I thought it was used in multiple places, but seems like isConnected is only used once. Will change it here.